docs/guide/sending-email.md

```diff
@@ -0,0 +1,226 @@
+
+Due to the difficulty of ensuring that emails are delivered reliably, we recommend that you utilize a delivery provider. However, if your server is set-up with both [DKIM](https://www.dkim.org/) and [SPF](https://en.wikipedia.org/wiki/Sender_Policy_Framework) ([DMARC](https://en.wikipedia.org/wiki/DMARC)) configured correctly, you can easily use the `sendmail` server app.
+
+## Configuring to use `sendmail`
+
+If your server has `sendmail` installed, `nodemailer` can use the service to send emails by setting the `sendmail` property to `true`. Both the `newline` and `path` properties may also need to be configured. By default, `newline` is set to `unix`(\<LF\> only), but it also accepts `windows`(\<LF\> and \<CR\>). The `path` property defaults to `usr/bin/sendmail`. If this isn't how your server is configured, then the correct path needs to be passed in.
+
+### Example
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  options: {
+    nodemailer: {
+      // enables the usage of `sendmail`
+      sendmail: true,
+      // changes the path to the `sendmail` app
+      path: 'user/sbin/sendmail'
+    }
+  }
+};
+
+```
+<template v-slot:caption>
+/modules/@apostrophecms/email
+</template>
+
+</AposCodeBlock>
+
+## Configuring using generic SMTP
+
+Using a 3rd-party email delivery provider like Gmail, or Mailgun will probably better guarantee delivery without the email being either bounced or sent to spam. Most providers use SMTP. In fact, the `sendmail` server application uses SMTP. It just performs most of the configuration for you automatically. To configure sending through other providers, you typically need to supply three different parameters to the `nodemailer` option through an object - the 'host', 'port', and 'auth' parameters. Depending on the host there are a number of other options that could be passed as outlined in the [`nodemailer` documentation](https://nodemailer.com/smtp/).
+
+### Example
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  options: {
+    nodemailer: {
+      host: "smtp.example.com",
+      port: 587,
+      auth: {
+        user: "username",
+        pass: "password"
+      }
+    }
+  }
+};
+
+```
+<template v-slot:caption>
+/modules/@apostrophecms/email
+</template>
+
+</AposCodeBlock>
+
+## Using specific service APIs
+
+The `nodemailer` app has three built-in transports - `sendmail`, `SES` - for sending by AWS SES, and `stream` - which is used for testing. You can create your own custom transport using the `nodemailer` [transport API documentation](https://nodemailer.com/plugins/create/#transports), or use one of the many transport plugins that are available. These can either be found on the providers site, or by [searching NPM](https://www.npmjs.com/search?q=nodemailer%20transport). Each transporter will have slightly different parameters configured through an object that is then passed along with the specific transport.
```

Comment:
"stream" sounds interesting, can it be used to achieve easy testing in a local development environment? If so can we show that?

Reply:
providers -> provider's